sql/core/src/test/scala/org/apache/spark/sql/internal/SQLConfSuite.scala

```diff
@@ -106,6 +114,16 @@ class SQLConfSuite extends QueryTest with SharedSparkSession {
       Row("spark.sql.groupByOrdinal", "false"))
   }
 
+  test("eager init") {
+    checkAnswer(
+      sql("SET").where(s"key = '${FakeConf.FAKE_CONFIG.key}'").select("key", "value"),
+      Nil)
```

Comment:
why this query has no result?

Reply:
```
    // (In Hive, "SET" returns all changed properties while "SET -v" returns all properties.)
    // Queries all key-value pairs that are set in the SQLConf of the sparkSession.
```
according to the comments in `SetCommand`, `SET` should not output the config that not changed.

Reply:
ah i see